### ISSUE TITLE
Windows folding arrows look bad

### ISSUE THREAD

**Issue by beruic** [guari/eclipse-ui-theme#1]
The folding arrows are still black on windows. Is this possible to fix?

**Comment by guari** [comment]
Hi beruic, unfortunately SWT uses system widgets and seems to not allow folding arrows customizations, maybe in future it will be provided some patch by the Eclipse Foundation. Currently the only way I know to change them on Windows is to modify them in system theme with some third party tool.

**Comment by beruic** [comment]
Ok. I just find it weird that they are white with black edges when I have white background, and completely black when I have black background. The overlay (blue) is fine when it's dark.
I have enclosed some screenshots so you can see what I mean. They do change color when changing to Moonrise, but they might be better off not doing this. At least on Windows.
![snap2](https://f.cloud.github.com/assets/724716/729347/9683c56a-e230-11e2-9eb7-7e03a0377397.png)
![snap1](https://f.cloud.github.com/assets/724716/729348/969f488a-e230-11e2-8d95-0033aa155a5f.png)

**Comment by guari** [comment]
I think the arrow is an image with alpha transparency, and it appears darker on dark background. The arrows are the same that you can find eg. in Windows file explorer.

**Comment by beruic** [comment]
Yeah. You're right. Let's just close this one for now, and hope that Eclipse at some point fixes this.

**Comment by paulvi** [comment]
Is Eclipse bug raised ? 

And I don't think this issue should be closed, as people keep asking (see #26 ).

**Comment by guari** [comment]
I've no idea if the related Eclipse bug is raised in the meantime. 
Till now, I've not had any feedbacks or updates for all the Eclipse bugs pointed out in this repo/project (so, taking the issues open or closed doesn't change a lot because they cannot be fixed here at plugin level).
You should poll the Eclipse Platform/UI team for that, but I think that those changes to SWT engine will require a rework of some parts of its logic since all the 'textured' widget are locked to inherit the corresponding OS widget (beside the fact that many of those widgets are not even exposed at CSS level, but this would be a minor issue).

Anyway if someone else has some news, glad to read them.

**Comment by paulvi** [comment]
I have raised [Bug 425646 - Support for black and inverted color UI themes](https://bugs.eclipse.org/bugs/show_bug.cgi?id=425646). It is definite that something is to be done on Eclipse platform side.

Could you please re-open the issue, so that it is not asked again.
